.circleci/config.yml

```diff
@@ -122,6 +122,16 @@ jobs:
     steps:
     - run: sudo apt install socat
     - checkout
+
+    - run:
+        git clone https://github.com/ipfs-shipyard/rb-pinning-service-api.git &&
+        sudo apt install ruby ruby-dev zlib1g-dev &&
+        sudo gem install rails
```

Comment:
I'm not sure why you need to install ruby and rails if you're running the service in docker-compose, it should include everything you need in it's Dockerfile.

Reply:
I don't know either. It seems to be required otherwise docker-compose fails. I am guessing rails has to come from somewhere into the Docker image ...

Reply:
Do you have the error message handy?